roles/edpm_nova/files/healthchecks/nova_compute/healthcheck

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/bash
+# -*- coding: utf-8 -*-
+# Copyright 2024 Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+pgrep -r DRST nova-compute
```

Comment:
we discussed this priviously in 


we discussed this previously in https://issues.redhat.com/browse/OSPRH-5007?focusedId=24196734&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-24196734

as perhaps a middle ground we could live with while haveing a check however I'm still not convinced we should do this or default to enabling health checks.

nova has  a per agent heartbeat which reports if the service si up.
systems and podman can both monidor if the contienr is running. so its not clear what value this is actlly adding.

Reply:
Yes I agree that in this case the health check does not bring additional value. IIRC we agreed on having this as until we will have a better solution. I can definitely remove nova_compute part until this happens.

Reply:
yes i forgot we agreed on thsi as a baseline

I'm still not particvally happy with external healthchecsk in general for any service but i can live with this. 

the tripleo version that tried to detect connections to rabbitmq and restart the container were actively harmful as it prevented oslo messaging trying to reconnect and nova form gracefully shutting down but this is much safer.